update_changelog.sh

```diff
@@ -16,7 +16,7 @@ fi
 # Remove the header so we can append the additions
 tail -n +3 "$CHANGELOG_FILE" > "$CHANGELOG_FILE.tmp" && mv "$CHANGELOG_FILE.tmp" "$CHANGELOG_FILE"
 
-docker run -it --rm -v "$(pwd)":/usr/local/src/your-app ferrarimarco/github-changelog-generator -u stargate -p stargate -t $GITHUB_TOKEN --since-tag $previous_version -b $CHANGELOG_FILE
+docker run -it --rm -v "$(pwd)":/usr/local/src/your-app ferrarimarco/github-changelog-generator -u stargate -p stargate -t $GITHUB_TOKEN --since-tag $previous_version --base $CHANGELOG_FILE --output $CHANGELOG_FILE --release-branch 'v1' --exclude-tags-regex 'v2.*' --exclude-labels 'stargate-v2,duplicate,question,invalid,wontfix'
```

Comment:
Ok this is good. For `main` we also have label `stargate-v1` in case we ever have v1-affecting only changes.

Reply:
Hmm, not sure if we have the `stargate-v1` label, do we?

Reply:
Yup, we do.